src/lib/seeds/EntityFactory.ts

```diff
@@ -81,4 +85,7 @@ export class EntityFactory<Entity> implements EntityFactoryInterface<Entity> {
         return entity;
     }
 
+    private isPromiseLike(object: any): boolean {
+        return object && typeof object.then === 'function';
```

Comment:
Please use this expression as it's more save:
`return !!object && (typeof object === 'object' || typeof object === 'function') && typeof object.then === 'function';`